sdktests/server_side_events_buffer.go

```diff
@@ -0,0 +1,91 @@
+package sdktests
+
+import (
+	"github.com/launchdarkly/sdk-test-harness/framework/ldtest"
+	"github.com/launchdarkly/sdk-test-harness/mockld"
+	"github.com/launchdarkly/sdk-test-harness/servicedef"
+
+	m "github.com/launchdarkly/go-test-helpers/v2/matchers"
+	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
+	"gopkg.in/launchdarkly/go-sdk-common.v2/ldvalue"
+	"gopkg.in/launchdarkly/go-server-sdk-evaluation.v1/ldbuilders"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func doServerSideEventBufferTests(t *ldtest.T) {
+	capacity := 20
+	eventsConfig := baseEventsConfig()
+	eventsConfig.Capacity = capacity
+
+	userFactory := NewUserFactory("doServerSideEventCapacityTests",
+		func(b lduser.UserBuilder) { b.Name("my favorite user") })
+	users := make([]lduser.User, 0)
+	for i := 0; i < capacity+3; i++ {
```

Comment:
I'm assuming the `+3` has to be any number > 0? Could give it a name like `const margin = 3` or something?